src/Seq.Api/SeqConnection.cs

```diff
@@ -14,12 +15,12 @@ public class SeqConnection : ISeqConnection
         readonly ConcurrentDictionary<string, Task<ResourceGroup>> _resourceGroups = new ConcurrentDictionary<string, Task<ResourceGroup>>();
         readonly Lazy<Task<RootEntity>> _root;
 
-        public SeqConnection(string serverUrl, string apiKey = null, bool useDefaultCredentials = true)
+        public SeqConnection(string serverUrl, string apiKey = null, bool useDefaultCredentials = true, CancellationToken token = default)
         {
             if (serverUrl == null) throw new ArgumentNullException(nameof(serverUrl));
             _client = new SeqApiClient(serverUrl, apiKey, useDefaultCredentials);
-            
-            _root = new Lazy<Task<RootEntity>>(() => _client.GetRootAsync());
+
+            _root = new Lazy<Task<RootEntity>>(() => _client.GetRootAsync(token));
```

Comment:
> Perhaps, since it's not an operation that's likely to take long anyway, we just ignore cancellation for now, here.

i think that is reasonable. want me to make the change?

Reply:
Sounds good, thanks :+1: